.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] [OPTIONAL] You can go ahead and add this file to your .gitignore file. Since this is simply a vscode settings file, you don't need to have it tracked in your repository